components/fieldgroups/profile/profile-consentResultsv2.schema.json

```diff
@@ -0,0 +1,107 @@
+{
+  "meta:license": [
+    "Copyright 2022 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/mixins/profile-consentResults",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Consent policies evaluation results map",
+  "type": "object",
+  "meta:intendedToExtend": [
+    "https://ns.adobe.com/xdm/context/profile"
+  ],
+  "description": "Consent policies evaluation result for individual",
+  "definitions": {
+    "profile-consentResultsV2": {
+      "properties": {
+        "xdm:consentPoliciesIDMapV2": {
```

Comment:
@mjasrotia is v2 accepted naming convention or should it be changed to something else?

Reply:
@ankushkd , having V2 kind of signifies that we would always have V1. Is there another suffix you guys can think of here ? How about "consentPoliciesMap" ?
cc: @vedhera

Reply:
I have renamed it to `consentPoliciesResultMap`. Although having v2 in the naming convention does not necessarily mean that v1 will always stick around. That's not how usually versioning works.

Reply:
@vedhera , as you mentioned above, I don't see the change in the name yet. Are we keeping v2 in the name ?